<?xml version="1.0"?>
<launch>
  <!-- cspell: ignore semseg, finetune  -->
  <!-- <arg name="image_path" default="$(env HOME)/autoware_data/000000000307.jpg" description="test img path"/> -->
  <arg name="image_path" default="/home/sc/work/autoware_lab/autoware/1719390321_839080583.jpg" description="test img path"/>
  <!-- <arg name="image_path" default="/home/sc/work/autoware_lab/autoware/zidane.jpg" description="test img path"/> -->
  <arg name="model_path" default="$(env HOME)/autoware_data/tensorrt_yolov10/yolov10m.onnx" description="test img path"/>

  <node pkg="autoware_tensorrt_yolov10" exec="yolov10_single_image_inference" name="tensorrt_yolov10_single_image_inference" output="screen">
    <param name="image_path" value="$(var image_path)"/>
    <param name="model_path" value="$(var model_path)"/>
  </node>
</launch>
